rust/types/src/markets.rs

```diff
@@ -69,6 +69,22 @@ pub struct Market {
     pub filters: MarketFilters,
 }
 
+impl Market {
+    /// returns the decimal places this market supports on the price
```

Comment:
```suggestion
    /// Returns the decimal places this market supports on the price.
```
Suggestion as this will make it into the documentation. (https://docs.rs/bpx-api-types/0.5.1/bpx_api_types/markets/struct.Market.html.)